src/components/SearchBar.js

```diff
@@ -1,14 +1,19 @@
-import React, { Component } from 'react'
-import ResultsList from './containers/ResultsList'
-import search from '../js/search'
```

Comment:
If anything, would you be able to insert that code into my working branch for me and then I'll reopen this PR?